src/Control/Apply.js

```diff
@@ -2,11 +2,14 @@
 
 exports.arrayApply = function (fs) {
   return function (xs) {
-    var result = [];
+    var l = fs.length;
+    var k = xs.length;
+    var result = Array(l*k);
```

Comment:
`new Array`? I know it's not a requirement for `Array`, I just find it super weird to construct without `new`.